website/src/css/_colors.scss

```diff
@@ -9,6 +9,12 @@ $brand-cyan: #00cbec;
 $brand-orange: #ff5543;
 $brand-primary-blue: #2a66e2;
 
+// Sourcegraph Brand Secondary Colors
+$brand-onahau: #c7ffff;
+$brand-mint: #8fedcf;
+$brand-fog: #e8d1ff;
```

Comment:
I actually use [name that colour](https://chir.ag/projects/name-that-color/#BFBFFF) as a colour db reference since our [brand guidelines lack some colour names](https://handbook.sourcegraph.com/departments/marketing/brand/brand_guidelines/color/). I figured we should start now and consolidate and then update the handbook as well in the interim while we wait for our brand/web designer to join our team. 🤓